src/features/query-bar/__tests__/__snapshots__/ValueField-test.js.snap

```diff
@@ -63,7 +63,7 @@ exports[`features/query-bar/components/filter/ValueField render value fields "sh
       id="boxui.queryBar.selectValuePlaceholderText"
     />
   }
-  selectedValue=""
+  selectedValue={null}
```

Comment:
This got updated because the placeholder only shows up when the selectedValue is null and not empty string.